.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "makefile.extensionOutputFolder": "./.vscode"
+}
```

Comment:
I have no idea about vscode, but this really seems more like a local preference that something that should be in the git repo.

Reply:
This is VSC specific. No idea if this should be part of the PR.

Reply:
No, it shouldn't. You can add `.vscode` into `~/.gitignore` (your user gitignore) and then add this ignore file to `~/.gitconfig`
```ini
[core]
	excludesfile = /path/to/user/.gitignore
``` 
This is the best place for host-specific and user-specific artifacts.